configs/edison/job_script.edison.bash

```diff
@@ -28,8 +28,8 @@ cd $SLURM_SUBMIT_DIR   # optional, since this is the default behavior
 export OMP_NUM_THREADS=1
 
 module unload python python/base
-module use /global/project/projectdirs/acme/software/modulefiles/all
-module load e3sm-unified/1.1.2
+source /global/project/projectdirs/acme/software/anaconda_envs/edison/base/etc/profile.d/conda.sh
```

Comment:
never mind. I forget that sometimes github doesn't show the updated file in comments..